lib/core/utils/main_navigation.dart

```diff
@@ -0,0 +1,29 @@
+import '../app_export.dart';
```

Comment:
avoid relative import, convert this to package import

Reply:
done